testcafe/helpers/navigations.js

```diff
@@ -2,6 +2,19 @@ import { Selector } from 'testcafe'
 
 import { createUserRole } from './roles'
 
+export const HOME_URL = '/v2/accueil'
+
+export const navigateToHomeAs = (user, userRole) => async t => {
+  const navBrandLogo = Selector('.nav-item').withText('Accueil')
```

Comment:
```suggestion
  const homepageNavItem = Selector('.nav-item').withText('Accueil')
```
Ou button, mais ce n'est plus le logo